Mathlib/RingTheory/Localization/Away/Basic.lean

```diff
@@ -8,6 +8,7 @@ import Mathlib.Algebra.Algebra.Pi
 import Mathlib.RingTheory.Ideal.Maps
 import Mathlib.RingTheory.Localization.Basic
 import Mathlib.RingTheory.UniqueFactorizationDomain.Multiplicity
+import Mathlib.Tactic.FieldSimp -- increase?
```

Comment:
I think we generally don't mind import increases caused by importing new tactics.